lib/lita/handlers/reviewme.rb

```diff
@@ -4,8 +4,10 @@ module Lita
   module Handlers
     class Reviewme < Handler
       REDIS_LIST = "reviewers"
+      DEFAULT_GITHUB_MSG = ":eyes: %{reviewer}"
 
       config :github_access_token
+      config :github_comment_template
```

Comment:
I noticed on https://docs.lita.io/plugin-authoring/configuration/, that the `config` class method accepts `:default` option that would be perfect for the default message!